server/config/options.go

```diff
@@ -17,6 +17,8 @@ package config
 import (
 	"time"
 
+	"github.com/tigrisdata/tigris/util"
+
```

Comment:
Remove new line between the imports

Reply:
Is there a linter / formatter where we could codify our preferences for imports? I got the opposite recommendation yesterday! :- ) I'm using `goimports` to format this file btw, not sure if that applies to whitespaces or not.

Reply:
@efirs do you have a preference for this?

Reply:
We use goimports. But it shouldn't have put a newline here. Newline is needed between new groups of imports.

Reply:
Yeah, combine them.
Basically we only want to separate Go's "builtin" packages into separate section. "time" in this case.

Reply:
goimports will automatically sort imports in the same group.